### DISCUSSION TITLE
Fortinet Logs

### DISCUSSION THREAD

**Discussion by mahmoudelhaj** [Security-Onion-Solutions/securityonion#11285] [2.4]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

20

### Storage for /

100

### Storage for /nsm

300

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

On 2.3 I was using the Filbeat module with a custom port and host group to ingest Forti logs.
With new SO 2.4 I and Elastic Agent 
I have added the Fortinet FortiGate Firewall Logs integration
& add portgroup, hostgroup but no logs show up
How to add agent on fortigate firewall integration to send logs to SO.
I am on a Distributed deployment.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/29997801/7c83f841-9e00-45ee-91f5-712eab607513)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/29997801/faf1c9c9-503d-4398-8219-ac2188940e3a)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/29997801/2f55f4ec-65cf-4f72-9cbd-0be10e8b3279)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by 8bitjoe** [comment]
It's not working atm, see this issue: https://github.com/Security-Onion-Solutions/securityonion/discussions/10942?sort=new
